src/Illuminate/Http/Client/Response.php

```diff
@@ -50,7 +50,7 @@ public function body()
     /**
      * Get the JSON decoded body of the response as an array.
```

Comment:
It might make sense to update the description to specify that the return type can vary too.

Reply:
Thanks for looking at this! What do you think the description should say?

`Get the JSON decoded body of the response as an associative array for objects and non-scalar types. Scalars will be mixed.` ?

I also was toying with the idea of checking if the decoded response was an array and if it wasn't wrapping then it in an array. My main concern would be possibly breaking others implementations.

Reply:
I don't think it makes sense to change the return value in any way, I think something like what you suggested for the description works, though it could probably be more brief.